apps/package.json

```diff
@@ -14,6 +14,7 @@
     "test:unit": "grunt unitTest",
     "test:integration": "grunt integrationTest",
     "test:entry": "grunt karma:entry",
+    "test:interpreter": "./node_modules/@code-dot-org/js-interpreter-tyrant/bin/run.js --run --diff --root ./test/interpreter --interpreter ./test/interpreter/patched-interpreter.js --progress",
```

Comment:
Because you list `"bin": "bin/run.js"` in [js-interpreter-tyrant's package.json](https://github.com/code-dot-org/js-interpreter-tyrant/blob/master/package.json#L6), it will be symlinked into [`node_modules/.bin`](https://docs.npmjs.com/files/package.json#bin), which is already [in the path when running an npm script](https://docs.npmjs.com/misc/scripts#path).  That means (I think) you can write this line as:
```
"test:interpreter": "run.js --run --diff --root ./test/interpreter --interpreter ./test/interpreter/patched-interpreter.js --progress",
```
...and probably should, as it's a little less configuration-dependent (it's possible customize the `node_modules` and `.bin` locations, though I'm not sure why you would).

This would be even nicer if you had js-interpreter-tyrant export a `js-interpreter-tyrant` executable instead, so it could be
```
"test:interpreter": "js-interpreter-tyrant --run --diff --root ./test/interpreter --interpreter ./test/interpreter/patched-interpreter.js --progress",
```
...as you describe [in the README for that tool](https://github.com/code-dot-org/js-interpreter-tyrant/blob/master/README.md#example-usage).